res/controllers/Pioneer-DDJ-400.midi.xml

```diff
@@ -506,7 +506,7 @@
             <control>
                 <description>RELOOP/EXIT (DECK1) - press - (loop off) Reloop, (loop on) Loop exit</description>
                 <group>[Channel1]</group>
-                <key>reloop_toggle</key> <!-- check if correct -->
+                <key>reloop_andstop</key> <!-- check if correct -->
```

Comment:
Why did you change this? I think `reloop_andstop` makes sense for the shifted press, but I don't understand why you are using it for the unshifted press too.

Reply:
By mistake. Fixed now.